web/src/components/dashboard/HideableTasks.tsx

```diff
@@ -0,0 +1,71 @@
+import { Icon } from "@/components/njwds/Icon";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useRoadmap } from "@/lib/data-hooks/useRoadmap";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { getSectionNames } from "@/lib/utils/helpers";
+import { LookupOperatingPhaseById } from "@businessnjgovnavigator/shared/operatingPhase";
+import { ReactElement } from "react";
+import { Button } from "../njwds-extended/Button";
+import { Step } from "../Step";
+import { SectionAccordion } from "./SectionAccordion";
+
+export const HideableTasks = (): ReactElement => {
+  const { userData, update } = useUserData();
+  const { roadmap } = useRoadmap();
+  const { Config } = useConfig();
+
+  if (!LookupOperatingPhaseById(userData?.profileData.operatingPhase).displayHideableRoadmapTasks) {
+    return <></>;
+  }
+
+  const handleToggleClick = () => {
+    if (!userData) return;
+    update({
+      ...userData,
+      preferences: {
+        ...userData.preferences,
+        isHideableRoadmapVisible: !userData.preferences.isHideableRoadmapVisible,
+      },
+    });
+  };
+
+  return (
+    <div className="margin-top-7" data-testid="hideableTasks">
+      <div className="fdr margin-bottom-105">
+        <h2>{Config.dashboardDefaults.upAndRunningTaskHeader}</h2>
+        <div className="mla">
+          <Button style="narrow-light" onClick={handleToggleClick}>
+            <div className="fdr fac">
+              <Icon>{userData?.preferences.isHideableRoadmapVisible ? "visibility_off" : "visibility"}</Icon>
+              <span className="margin-left-05 line-height-sans-2">
+                {userData?.preferences.isHideableRoadmapVisible
+                  ? Config.dashboardDefaults.hideTaskText
+                  : Config.dashboardDefaults.showTaskText}
```

Comment:
should we use a MUI Accordion for this? That way we get the nice animation on the show/hide movement like we do for sections in the roadmap

Reply:
keeping it as is for now, we may create a design qa ticket if Cary wants the accordion animation